src/Microsoft.ML.SamplesUtils/ConsoleUtils.cs

```diff
@@ -35,5 +36,15 @@ public static void PrintMetrics(RegressionMetrics metrics)
             Console.WriteLine($"RMS: {metrics.Rms:F2}");
             Console.WriteLine($"RSquared: {metrics.RSquared:F2}");
         }
+
+        /// <summary>
+        /// Pretty-print RankerMetrics objects.
+        /// </summary>
+        /// <param name="metrics">Ranker metrics.</param>
+        public static void PrintMetrics(RankerMetrics metrics)
+        {
+            Console.WriteLine($"DCG@N: {string.Join(", ", metrics.Dcg.Select(d => Math.Round(d, 2)).ToArray())}");
```

Comment:
>@N [](start = 35, length = 2)

'N' should be an actual number. @1 @3 @5 etc. we should include that in the printing, otherwise people won't know what the numbers mean.